open_rarity/resolver/opensea_api_helpers.py

```diff
@@ -26,7 +33,7 @@
 OS_METADATA_TRAIT_TYPE = "display_type"
 
 
-def fetch_opensea_collection_data(slug: str):
+def fetch_opensea_collection_data(slug: str) -> dict:
```

Comment:
Any value in making this a `TypedDict`

Reply:
I will leave as is as this is really not the core library but just used to compare ranking data across providers